sil-dictionary.php

```diff
@@ -102,10 +92,8 @@
 
 add_filter('posts_request','replace_default_search_filter');
 
-if( !is_page())

```

Comment:
WordPress was complaining about calling `is_page()` in this context. Moved it to `my_enqueue_css()` in `dictionary-search.php`.